index.bs

```diff
@@ -1,17 +1,84 @@
 <pre class='metadata'>
-Title: Storage Buckets
+Title: Storage Buckets API
 Shortname: storage-buckets
 Level: 1
 Status: CG-DRAFT
 Group: WICG
 Repository: WICG/storage-buckets
-URL: http://example.com/url-this-spec-will-live-at
-Editor: Victor Costan, Google Inc. https://google.com, costan@google.com
-!Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/storage-buckets>web-platform-tests storage-buckets/</a> (<a href=https://github.com/w3c/web-platform-tests/labels/storage-buckets>ongoing work</a>)
-Abstract: A short description of your spec, one or two sentences.
+URL: https://wicg.github.io/storage-buckets/
+Editor: Evan Stade, Google https://www.google.com/, estade@google.com
+Editor: Ayu Ishii, Google https://www.google.com/, ayui@google.com
+Former Editor: Victor Costan
+!Participate: <a href="https://github.com/WICG/storage-buckets">GitHub WICG/storage-buckets</a> (<a href="https://github.com/WICG/storage-buckets/issues/new">new issue</a>, <a href="https://github.com/WICG/storage-buckets/issues?state=open">open issues</a>)
+Abstract: The storage buckets API provides a way for a site to create multiple storage buckets, where the user agent may choose to delete each bucket independently of other buckets.
```

Comment:
Perhaps:

The Storage Buckets API provides a way for sites to organize locally stored data into groupings called "storage buckets". This allows the user agent or sites to manage and delete buckets independently rather than applying the same treatment to all the data from a single origin.

Reply:
This is better, thanks!!